apps/play/src/components/knobs/ParamKnob.module.css

```
@@ -0,0 +1,34 @@
+.knobContainer {
+  --knob-font-size: 12px;
+  --knob-el-width: 45px;
+
+  line-height: 1.5;
+  font-size: var(--knob-font-size);
+
+  display: flex;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)